tests/test_persistence.py

```diff
@@ -476,12 +476,22 @@ def test_migrate_old(mocker, default_conf, fee):
                                      stake=default_conf.get("stake_amount"),
                                      amount=amount
                                      )
+    insert_table_old2 = """INSERT INTO trades (exchange, pair, is_open, fee,
+                          open_rate, close_rate, stake_amount, amount, open_date)
+                          VALUES ('BITTREX', 'BTC_ETC', 0, {fee},
+                          0.00258580, 0.00268580, {stake}, {amount},
+                          '2017-11-28 12:44:24.000000')
+                          """.format(fee=fee.return_value,
+                                     stake=default_conf.get("stake_amount"),
+                                     amount=amount
+                                     )
```

Comment:
did not get this piece
(however, I'm not a db-geek, assume this is okay)

Reply:
well to test if `close_profit_abs` is calculated correctly during database migration (and aligned to the python calculation), we need a closed trade too - since it'll not calculate `close_profit_abs` for open trades.

Reply:
I mean why it's introduced as a sep. insert_table_old2 if it's executed sequentially:
```
    engine.execute(insert_table_old)
    engine.execute(insert_table_old2)
```
?

Reply:
because it's cleaner than using 2 statements in one execute - and because `insert into values ()` does not support multiple rows (at least not on all databases)